apps/mdn/mdn-aws/k8s/Makefile

```diff
@@ -235,6 +235,7 @@ export NEW_RELIC_WEB_NAME ?= "kuma-web-${NEW_RELIC_MONITOR_SUFFIX};${NEW_RELIC_R
 export NEW_RELIC_API_NAME ?= "kuma-backend-${NEW_RELIC_MONITOR_SUFFIX};${NEW_RELIC_ROLLUP_NAME}"
 export NEW_RELIC_KUMASCRIPT_NAME ?= "kumascript-${NEW_RELIC_MONITOR_SUFFIX}"
 export NEW_RELIC_CELERY_NAME ?= "kuma-backend-${NEW_RELIC_MONITOR_SUFFIX};${NEW_RELIC_ROLLUP_NAME}"
+export NEW_RELIC_SSR_NAME ?= "kuma-ssr-${NEW_RELIC_MONITOR_SUFFIX};${NEW_RELIC_ROLLUP_NAME}"
```

Comment:
I was originally thinking that this would be just `ssr-${NEW_RELIC_MONITOR_SUFFIX}` (with no roll-up, like `kumascript`, since it's a service similar to that one), but your idea here made me doubt that, and that perhaps it's better to keep it organized by Docker image like this.